test/parallel/test-whatwg-encoding-textdecoder-ignorebom.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+// From: https://github.com/w3c/web-platform-tests/blob/7f567fa29c6805c92f5b63590338ccb041aacf67/encoding/textdecoder-ignorebom.html
+
+const common = require('../common');
+
+if (!common.hasIntl)
+  common.skip('missing Intl');
```

Comment:
With the exception of UTF-16BE, this test should work even when ICU is not enabled.